.github/dependabot.yaml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+    labels:
+      - ci
+      - github-actions
+      - dependencies
+  - package-ecosystem: pip
+    directory: /
+    schedule:
+      interval: weekly
+    labels:
+      - ci
+      - pip
+      - dependencies
```

Comment:
Given that Kodi doesn't use the pip ecosystem and uses it's own dependency system, I'm not sure that dependabot actually makes sense here?  Unless it has hooks into Kodi's addon repos somehow.  Or I'm missing something.

Reply:
Well, in this case dependabot picks up on the `requirements-dev.txt` and monitors that one. (If you prefer I will limit it to github-actions only)

Reply:
That makes sense, I had forgot about that file honestly.  Carry on.